From 3771b357d8a0d84af0405c505043bde7fee6ccb8 Mon Sep 17 00:00:00 2001
From: Jeremie Graulle <jeremiex.graulle@intel.com>
Date: Tue, 2 Apr 2013 13:05:52 +0200
Subject: libmedia_helper: Host static lib compil for UT

BZ: 93434
[CLEAN]

Need to compile this project in host static lib mode
in order to compile system and unit test of the
"Configurable HAL" component.

Add a host static lib compilation in libmedia Android.mk.

Category: aosp improvement
Domain: AUDIO_MEDIA-Audio Framework
Origin: internal
Upstream-Candidate: no, proprietary
Change-Id: Id64b35199aa6ad08356b12b246ebcf472d9c9236
Orig-Change-Id: Ibf30b297abc4d9512bf4d9154c411b18c7147371
Signed-off-by: Jeremie Graulle <jeremiex.graulle@intel.com>
---
 media/libmedia/Android.mk |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)

diff --git a/media/libmedia/Android.mk b/media/libmedia/Android.mk
index 56e7787..e05c88b 100644
--- a/media/libmedia/Android.mk
+++ b/media/libmedia/Android.mk
@@ -1,5 +1,8 @@
 LOCAL_PATH:= $(call my-dir)
 
+# Build for helper target
+#########################
+
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
@@ -9,6 +12,21 @@ LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_STATIC_LIBRARY)
 
+# Build for helper host test
+############################
+
+include $(CLEAR_VARS)
+
+LOCAL_SRC_FILES:= \
+    AudioParameter.cpp
+LOCAL_MODULE:= libmedia_helper_host
+LOCAL_MODULE_TAGS := optional
+
+include $(BUILD_HOST_STATIC_LIBRARY)
+
+# Build for main lib target
+###########################
+
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
-- 
1.7.4.1

